packages/connect-core/src/entities/Organization.ts

```diff
@@ -125,14 +154,83 @@ export default class Organization {
   }
 
   ///////// INTENTS ///////////
-  appIntent(
-    appAddress: Address,
-    functionName: string,
-    functionArgs: any[]
-  ): TransactionIntent {
-    return new TransactionIntent(
-      { contractAddress: appAddress, functionName, functionArgs },
-      this,
+
+  /**
+   * Calculate the transaction path for a transaction to an external `destination`
+   * (not the currently running app) that invokes a method matching the
+   * `methodAbiFragment` with `params`.
+   *
+   * @param  {string} destination Address of the external contract
+   * @param  {object} methodAbiFragment ABI fragment of method to invoke
+   * @param  {Array<*>} params
+   * @param  {Object} options
+   * @return {Promise<ForwardingPath>} An object that represents the forwarding path corresponding to an action.
+   *   If the destination is a non-installed contract, always results in a
+   *   single step transaction.
+   */
+  async intent(
```

Comment:
As discussed in the API spec we update the `appIntent` method to become the `intent` method. But after further discussions offline we agree that possible is a good idea to just remove the intent all together from the Organization. Do you agree?